dbms/src/Server/Server.cpp

```diff
@@ -874,6 +685,8 @@ int Server::main(const std::vector<std::string> & /*args*/)
         storage_config.s3_config.isS3Enabled());
 
     const auto is_compute_mode = global_context->getSharedContextDisagg()->isDisaggregatedComputeMode();
+    const auto is_storage_mode = global_context->getSharedContextDisagg()->isDisaggregatedStorageMode();
```

Comment:
```suggestion
    const auto is_disagg_compute_mode = global_context->getSharedContextDisagg()->isDisaggregatedComputeMode();
    const auto is_disagg_storage_mode = global_context->getSharedContextDisagg()->isDisaggregatedStorageMode();
```
Adding `disagg` to the name is less ambiguous